### ISSUE TITLE
Biolink prefix confusion, inconsistencies: how do we represent specific kinds of variants, and which identifier namespaces are preferred.

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#54]
The QotM raised a number of issues related to Biolink prefixes, several of which may be worth further investigation.

**Variant IDs (and any cross-mapping)**

- [CLINVAR:1333833](https://www.ncbi.nlm.nih.gov/clinvar/variation/1333833/?oq=1333833&m=NM_152296.5(ATP1A3):c.2552A%3EG%20(p.Gln851Arg)) or ClinVarVariant:1333833?

_It looks like both are listed as valid prefixes in Biolink, but only CLINVAR is in the context.jsonld, so we probably need to remove ClinVarVariant? Probably a good biolink issue._

- [DBSNP:rs869320661](https://www.ncbi.nlm.nih.gov/snp/rs869320661)

_Biolink considers rs ids a poor match for variants - they are really more of a locus id - though everybody uses them like this._

- [CAID:CA358802](http://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA358802) or remove the "CA" part?

_Keep "CA" b/c then the url resolves correctly. FWIW, the CAID is really meant to function like a node normalizer for variants, providing all the other names for that entity (in this case including variations across different versioning of the underlying sequences)._

- Thoughts on curies for genomic deletions / CNVs?

_For small deletions CAIDs should work; for bigger and for CNV IMO there is not a perfect approach._

**Disease IDs:** 

- Is the biolink-model prefix ORPHANET or [ORPHA:2131](https://hpo.jax.org/app/browse/disease/ORPHA:2131)?

_Looks like it should be ORPHANET, but I am confused as well about why we have both in different places._

**Chemicals:**

- What are the CURIES for cannabidiol, valproic acid–induced hepatotoxicity, hepatocyte depolarization/polarity.

**Comment by gglusman** [comment]
"Biolink considers rs ids a poor match for variants - they are really more of a locus id"
I'm not sure I understand the issue here. A variant refers to variation at a locus, and rsids are excellent for identifying variants. Is there a mixup with allele - the specific version observed for that variant?

**Comment by sierra-moxon** [comment]
@gglusman - the plan is to do a re-work of allele/variant in biolink in 2023 for sure.  rs ids are a good identifier for polymorphisms for sure.

**Comment by karafecho** [comment]
Also see:

https://github.com/biolink/biolink-model/issues/1119;
https://github.com/biolink/biolink-model/issues/1121; [https://github.com/biolink/biolink model/issues/1122](https://github.com/biolink/biolink%20model/issues/1122)

**Comment by sstemann** [comment]
@sierra-moxon can you review this ticket?  i'm not sure if it's still open?

**Comment by sierra-moxon** [comment]
@karafecho - I reviewed all the related Biolink tickets tagged here and I believe I've answered all the questions with what Biolink (and bioregistry by extension) thinks the prefixes for these IDs should be.  I do not have a way to validate that all the KPs are using the correct prefixes in bulk, but I do believe the reasoner-validator will warn of incorrect prefix use.  

Can we close this ticket?
